DashAI/front/src/components/experiments/ConfigureModelsStep.jsx

```diff
@@ -14,25 +15,53 @@ import uuid from "react-uuid";
  * @param {function} setNewExp updates the Eperimento Modal state (newExp)
  * @param {function} setNextEnabled function to enable or disable the "Next" button in the modal
  */
+
+const compatibleModelsTest = [
+  { name: "KNeighborsClassifier" },
+  { name: "RandomForestClassifier" },
+  { name: "SVC" },
+  { name: "numericalWrapperForText" },
+  { name: "tcTransformerEngSpa" },
+];
+
 function ConfigureModelsStep({ newExp, setNewExp, setNextEnabled }) {
   const { enqueueSnackbar } = useSnackbar();
   const [modelNickname, setModelNickname] = useState("");
   const [selectedModel, setSelectedModel] = useState("");
-
-  const compatibleModels = [
-    "KNeighborsClassifier",
-    "RandomForestClassifier",
-    "SVC",
-    "numericalwrapperfortext",
-    "tcTransformerEngSpa",
-  ];
+  const [compatibleModels] = useState(compatibleModelsTest);
 
   // width for model nickname and model type textfields
   const textFieldWidth = "32vw";
 
+  // const getCompatibleModels = async () => {
+  //   try {
+  //     const models = await getComponentsRequest({
+  //       selectTypes: ["Model"],
+  //       relatedComponent: newExp.task_name,
+  //     });
+  //     setCompatibleModels(models);
+  //   } catch (error) {
+  //     enqueueSnackbar("Error while trying to obtain compatible models", {
+  //       variant: "error",
+  //       anchorOrigin: {
+  //         vertical: "top",
+  //         horizontal: "right",
+  //       },
+  //     });
+  //     if (error.response) {
+  //       console.error("Response error:", error.message);
+  //     } else if (error.request) {
+  //       console.error("Request error", error.request);
+  //     } else {
+  //       console.error("Unkown Error", error.message);
+  //     }
+  //   }
+  // };
+
```

Comment:
I think this should be de-commented.

Reply:
Yes, i was using a test version of the data to be able to test all the models